From 7884ad74f2c0bbd568eee1fa1fefdb27ddd4ef5c Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Fri, 13 May 2016 15:27:43 +0100
Subject: [PATCH] Enforce ABI versioning, similar to Fedora patch

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 googletest/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/googletest/CMakeLists.txt b/googletest/CMakeLists.txt
index 9ee7940..9af2598 100644
--- a/googletest/CMakeLists.txt
+++ b/googletest/CMakeLists.txt
@@ -131,7 +131,9 @@ endif()
 # are used for other targets, to ensure that gtest can be compiled by a user
 # aggressive about warnings.
 cxx_library(gtest "${cxx_strict}" src/gtest-all.cc)
+set_target_properties(gtest PROPERTIES VERSION 0.0.0 SOVERSION 0)
 cxx_library(gtest_main "${cxx_strict}" src/gtest_main.cc)
+set_target_properties(gtest_main PROPERTIES VERSION 0.0.0 SOVERSION 0)
 # If the CMake version supports it, attach header directory information
 # to the targets for when we are part of a parent build (ie being pulled
 # in via add_subdirectory() rather than being a standalone build).
-- 
2.19.0

